lib/factree.rb

```diff
@@ -1,5 +1,9 @@
 require "factree/version"
+require "factree/conclusion"
+require "factree/path"
+require "factree/aggregate"
+require "factree/dsl"
 
 module Factree
-  # Your code goes here...
+  extend DSL
```

Comment:
This makes all of the `DSL` methods accessible as class methods on `Factree`, so you can access them that way if you don't want to `extend DSL` in your own code.